### ISSUE TITLE
Expo Plugin Support

### ISSUE THREAD

**Issue by skyshocker** [Hector-Chong/native-wechat#17]
Hi,

Thanks for creating such new library for supporting WeChat.
However, A lot of new Expo projects are using managed workflow , which doesn't use prebuild to 'eject' the project so there is no way to modify the native codes. Expo-plugin is a feature to wrap all of those native changes so that projects with managed workflow don't have to modify the native codes.

Could you please consider adding such expo-plugin support to your project?
It will be a lot helpful for modern projects.

Thanks.

**Comment by xixilive** [comment]
As an alternative，you can fork this gist: https://gist.github.com/xixilive/204e8a1bae02d6edc0f0d6337d8295c2 @skyshocker

**Comment by Hector-Chong** [comment]
> As an alternative，you can fork this gist: https://gist.github.com/xixilive/204e8a1bae02d6edc0f0d6337d8295c2 @skyshocker

Thanks, that's a great help! I'll try that later.

**Comment by XHFkindergarten** [comment]
I have been looking for expo wechat module for years 😆
If there is some progress on this topic, it will be of great help to my project.
(Is it convenient for you to share the appreciation channels at that time)

**Comment by Hector-Chong** [comment]
> I have been looking for expo wechat module for years 😆 If there is some progress on this topic, it will be of great help to my project. (Is it convenient for you to share the appreciation channels at that time)

As I've been reassigned, I'm no longer in charge of WeChat-related SDK development. If you could provide your contact information and necessary parameters, including appid, universal URL, signature for APK and so on, to my email, it would be great help for me make it compatible with Expo.

Regrading the appreciation channels, well, a star for this project would be just perfect.

**Comment by XHFkindergarten** [comment]
> > I have been looking for expo wechat module for years 😆 If there is some progress on this topic, it will be of great help to my project. (Is it convenient for you to share the appreciation channels at that time)
> 
> As I've been reassigned, I'm no longer in charge of WeChat-related SDK development. If you could provide your contact information and necessary parameters, including appid, universal URL, signature for APK and so on, to my email, it would be great help for me make it compatible with Expo.
> 
> Regrading the appreciation channels, well, a star for this project would be just perfect.

yeah, of course. I will email you my parameters as soon as possible, thank you for your generosity.

**Comment by dwcooper** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
jwma, @Running-Chen911, @tjgljs, @dongdong-gogogo, @zgyanglinjie, @wj0s3ph, @dianxiang-sun, @8U61ife, @salemalem, @liu-shilong, @0xsongsu, @kingzheng, @Sicmatr1x, @mhquant, @CoderPolyglot, @pi-pi-miao, @a1740942002, @Lnek, @wkler, @ZhangLei012

**Comment by Hector-Chong** [comment]
Definitely fishing. Really funny.

**Comment by pi-pi-miao** [comment]
> > Hello,
> > We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!
> > As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:
> > 
> > * Health insurance coverage
> > * Retirement savings plan
> > * Flexible work schedule
> > * Generous vacation and paid time off
> > * Professional development opportunities
> > 
> > To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.
> > Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.
> > **Important:** You have 24 hours to complete the application process.
> > If you have any questions or need further information, please don't hesitate to contact us.
> > Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.
> > Best regards, GitHub Recruitment Team jwma, @Running-Chen911, @tjgljs, @dongdong-gogogo, @zgyanglinjie, @wj0s3ph, @dianxiang-sun, @8U61ife, @salemalem, @liu-shilong, @0xsongsu, @kingzheng, @Sicmatr1x, @mhquant, @CoderPolyglot, @pi-pi-miao, @a1740942002, @Lnek, @wkler, @ZhangLei012
> 
> Email fishing?

I don't know, but I applied
